x-pack/plugins/infra/public/components/subscription_splash_content.tsx

```diff
@@ -103,16 +96,59 @@ export const SubscriptionSplashPrompt: React.FC = () => {
     );
   }
 
+  return (
+    <PageTemplate {...templateProps} isEmptyState>
+      <EuiEmptyPrompt
+        iconType={'visLine'}
+        title={<h2>{title}</h2>}
+        body={
+          <EuiText>
+            <p>{description}</p>
+          </EuiText>
+        }
+        actions={cta}
+      />
+    </PageTemplate>
+  );
+};
+
+export const SubscriptionSplashPrompt: React.FC = () => {
+  const manageLicenseURL = useLicenseUrl();
+  const { services } = useKibanaContextForPlugin();
+
   return (
     <EuiEmptyPrompt
-      iconType={'visLine'}
-      title={<h2>{title}</h2>}
+      iconType="logoObservability"
+      iconColor="warning"
+      title={
+        <h3>
+          {i18n.translate('xpack.infra.ml.splash.trial.title', {
+            defaultMessage:
+              'Discover and resolve infrastructure issues faster with anomaly detection',
+          })}
+        </h3>
+      }
       body={
-        <EuiText>
-          <p>{description}</p>
-        </EuiText>
+        <p>
+          {i18n.translate('xpack.infra.ml.splash.trial.description', {
+            defaultMessage:
+              'Uncover infrastructure anomalies to preempt issues and resolve them quicker with the aid of machine learning.',
+          })}
+        </p>
       }
-      actions={cta}
+      actions={[
+        <RedirectAppLinks coreStart={services}>
```

Comment:
I wonder why we don't wrap the entire infra app with this. Not in scope of this PR though.

Reply:
Hmmm good question, I saw we used that multiple times for different links but it's better to do it like APM [here](https://github.com/elastic/kibana/blob/591d2512e2ef4ba0c7756af6426e6ad17260a2be/x-pack/plugins/apm/public/components/routing/app_root/index.tsx#L70) - I will add another issue for that as part of the infra code improvement/consistency with APM series.
cc @crespocarlos fyi

Reply:
The issue is added to the backlog ([#177056](https://github.com/elastic/kibana/issues/177056))

Reply:
Thank you!!